examples/ex71.hpp

```diff
@@ -0,0 +1,587 @@
+// shared implementation ex71p/ex71 for the AD integrands and
+// the handconded integrators
+
+
+#ifndef EXAMPLE71_H
+#define EXAMPLE71_H
+
+#include "mfem.hpp"
+#include <memory>
+#include <iostream>
+#include <fstream>
+
+
+namespace mfem
+{
+
+
+template<typename DType, typename MVType>
```

Comment:
I assume these are data types and vector types?

Reply:
Yes they are. The new code is commented.